scripts/cddl_gen.py

```diff
@@ -1525,6 +1525,8 @@ def parse_args():
                         help="Path to output C file.")
     parser.add_argument("--output-h", "--oh", required=True, type=str,
                         help="Path to output header file.")
+    parser.add_argument("--output-h-types", "--oht", required=False, type=str,
```

Comment:
Could this use a more sophisticated type to ensure that its possible to write to the file etc?

Reply:
Sounds like a good idea. I'll have to look into it a bit.

Reply:
done